azext_iot/sdk/iothub/pnp_runtime/__init__.py

```diff
@@ -0,0 +1,18 @@
+# coding=utf-8
```

Comment:
i thought we were moving away from having sdks like this, or was it just the `_bind_sdk` call that we are trying to mover away from

Reply:
still want to use the sdk's and yea removal of `_bind_sdk` is the goal -- outdated/legacy.